QtCollider/QcCallback.hpp

```diff
@@ -0,0 +1,64 @@
+/*
+    SuperCollider Language
+    Copyright (c) 2018 SuperCollider Team
+    https://supercollider.github.io/
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program; if not, write to the Free Software
+    Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301  USA
+*/
+
+#pragma once
```

Comment:
haha, i don't get it either. turns out it's not actually used outside of WebView, something i should have noticed prior to refactoring.

Reply:
So does that mean you can move it back to where it was, or is the plan to wrap it inside of a WEBENGINE #ifdef, or? Also I still think a comment would be helpful here.

Reply:
the simple answer is that i don't think it's worth the effort to go back and undo this -- although it wasn't necessary for me to refactor this part, it doesn't make the structure any worse.

i looked a bit into where it's used and it seems to be a way of connecting sclang functions to Qt callbacks. that seems reasonably generic, and there's a chance that someone would want to use it outside of WebView.

i'd like to add a comment but i'm not sure what i could add that would be insightful, since, as i said, i don't have a great understanding of what this code does.